README.md

```diff
@@ -5,58 +5,55 @@
 
 For a more detailed overview of IoT Edge Dev Tool including setup and commands, please see the [Wiki](https://github.com/Azure/iotedgedev/wiki).
 
 ## Quickstart
 
-> To set up development machines manually instead of using the IoT Edge Dev Container, please see the [Manual Development Machine Setup Wiki](https://github.com/Azure/iotedgedev/wiki/manual-dev-machine-setup).
-
 This quickstart will run a container, create a solution, setup Azure resources, build and deploy modules to your device, setup and start the IoT Edge simulator, monitor messages flowing into IoT Hub, and finally deploy to the IoT Edge runtime.
 
-<!-- Here's a 3 minute video walk-through of this Quickstart:
+At the end of this quickstart, you should have the following:
 
-[![Azure IoT Edge Dev Tool: Quickstart](assets/edgedevtoolquickstartsmall.png)](https://aka.ms/iotedgedevquickstart) -->
+- Development environment.
+- IoT Hub deployed in your Azure subscription.
+- A directory with the [IoT edge deployment manifest](https://docs.microsoft.com/en-us/azure/iot-edge/module-composition?view=iotedge-2020-11), [.env file](.env.tmp) with the environment variables needed for the `iotedgedev` tool, and the source file of a custom IoT edge module (`filtermodule`).
+- The docker image of the `filtermodule`.
+- Starting the IoT Edge simulator, which runs the containers defined in the device manifest.
+- Monitoring the messages sent to the IoT Hub.
+- Cleaning up the docker containers and images.
 
-The only thing you need to install is Docker. All of the other dev dependencies are included in the container. 
+### Pre-requisites
 
-1. **Install [Docker CE](https://docs.docker.com/install/)**
+- Docker (see [Install Docker docs](./docs/Install-Docker.md) for details).
 
-    - For Windows, please follow [the document](https://docs.docker.com/docker-for-windows/#shared-drives) to open Docker Settings and setup a Shared Drive.
-    - For macOS, please follow [the document](https://docs.docker.com/docker-for-mac/#file-sharing) to choose local directories to share with your containers.
-    
-    Note: If the device is behind the proxy server, you can set the [proxy manually](https://docs.docker.com/network/proxy/#configure-the-docker-client) 
+### Quickstart steps
 
-1. **Run the IoT Edge Dev Container**
+1. Setup development environment
 
-    Before you run the container, you will need to create a local folder to store your IoT Edge solution files.
-    
-    **Windows**
-    ```
-    mkdir c:\temp\iotedge
-    docker run -ti -v /var/run/docker.sock:/var/run/docker.sock -v c:/temp/iotedge:/home/iotedge mcr.microsoft.com/iotedge/iotedgedev
-    ```
+    There are three options to setup your development environment:
 
-    **Linux**
-    ```
-    sudo mkdir /home/iotedge
-    sudo docker run -ti -v /var/run/docker.sock:/var/run/docker.sock -v ~/iotedge:/home/iotedge mcr.microsoft.com/iotedge/iotedgedev

```

Comment:
>mcr.microsoft.com/iotedge/iotedgedev [](http://example.com/codeflow?start=96&length=36)

Are you recommending not to sue the mcr image?

Reply:
why do you need to run the docker command as root?

Reply:
> why do you need to run the docker command as root?

sudo is needed if the current user is not a member of the 'docker' group

Reply:
sure, but you usually install docker and add the corresponding user to the docker group

Reply:
We can drop the 'sudo' if we tell the user to add the corresponding user to the docker group. Users should be reminded of that step...